src/haxeLanguageServer/features/CompletionFeature.hx

```diff
@@ -36,7 +36,7 @@ class CompletionFeature {
     }
 
     static var reFieldPart = ~/(\.|@(:?))(\w*)$/;
-    static var reStructPart = ~/[(,]\s*{\s*((\s*\w+\s*:\s*["'\w()\.]+\s*,\s*)*\w*)$/;
+    static var reStructPart = ~/[(,]\s*{(\s*(\s*\w+\s*:\s*["'\w()\.]+\s*,\s*)*\w*)$/;
```

Comment:
I had to move the `\s` into the capture group for the whitespace fix to actually work.